### PR TITLE
feat(save): add "copy" button

### LINKED ISSUE #49: Feature request: Copy Image to clipboard
It will be really great if there is copy to clipboard option.
Quite helpful when you have to share the image to slack - discord or any other chatting application which supports direct upload.

### PR DESCRIPTION
**Pull request by VinSpee**
add a button to the save menu called "copy", which opens the raw image data in the window so users
can directly copy it for easy pasting into a variety of apps

Closes #49

There is currently not browser support for copying actual images, so this seems to be a reasonable workaround. Any thoughts?

jul-25-2018 15-03-11

### CODE REVIEW

File: components/Editor.js [hunk 1]
```diff
@@ -146,6 +146,10 @@ class Editor extends React.Component {
     const timestamp = this.state.timestamp ? `_${formatTimestamp()}` : ''
 
     return this.getCarbonImage({ format, type: 'blob' }).then(url => {
+      if (format === 'copy') {
+        window.location = url;
```
Comment (mfix22) [inline components/Editor.js]: Is there a way you can open it in a new tab? I think you can do this similar to how we handle the download case, but without setting "download".

File: components/Editor.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ const saveButtonOptions = {
   button: true,
   color: '#c198fb',
   selected: { id: 'SAVE_IMAGE', name: 'Save Image' },
-  list: ['png', 'svg'].map(id => ({ id, name: id.toUpperCase() }))
+  list: ['png', 'svg', 'copy'].map(id => ({ id, name: id.toUpperCase() }))
```
Comment (mfix22) [inline components/Editor.js]: can you call the button 'open ↗`

**Comment by mfix22** [comment]
@VinSpee are you able to update this PR based on the comments above? 🙂

**Comment by VinSpee** [comment]
Ooooh yep! I can today!

**Comment by VinSpee** [comment]
👍🏻 check it out

**Comment by mfix22** [comment]
With `target: "_blank"` ad-blockers will sometimes block URLs. We can either live with that, or just remove `target: "_blank"` (and probably the `↗` with it?). 

@briandennis @jakedex thoughts?

**Pull request merged**